src/_guides/language/evolution.md

```diff
@@ -270,6 +270,22 @@ To learn more about named parameters and arguments, see the
 
 ### Dart 2.18
 
+Dart 2.18 enhanced type inference by allowing information flow between arguments in generic function calls.
+Leaving the argument type unspecified within some invocations previously triggered type errors,
+citing potential null occurrences.
+The compiler can now infer the argument type from other values in an invocation,
+so specifying the argument type inline is no longer necessary. 
+For more information,
+see [Type argument inference][].
```

Comment:
**suggestion:** Simplify the copy. Sometimes, it's better to cut sentences into shorter, linked ideas.
```suggestion
Dart 2.18 enhanced type inference. This change allows information flow between arguments in generic function calls. Before 2.18, if you didn't specify an argument's type in some methods, Dart returned errors. These type errors cited potential null occurrences. With 2.18, the compiler infers the argument type from other values in an invocation. You don't need to specify the argument type inline. To learn more, see [Type argument inference][].
```

As written, this is Grade 14 readability at 63 words. As revised, it's Grade 7 with 71 words. Clear, concise copy should be the goal. 

**issue:** This may be personal preference, but I really have an issue with "For more information..." or "For more details...". This phrasing removes agency from the reader. I prefer "To learn more...". This serves as a call to action.